wiki/Tournaments/TEMPLATE.md

```diff
@@ -0,0 +1,159 @@
+---
+tags:
+  - {SHORT_TITLE}
+  - {SERIES_SHORT_TITLE}
+---
+
+# {TITLE}
+
+![{SHORT_TITLE} logo](img/logo.png) <!-- replace "logo" with "banner" if that suits the image better -->
+
+The **{TITLE}** (***{SHORT_TITLE}***) is a <!-- enter applicable tournament traits, example: {N}v{N}, double-elimination, country-based --> {MODE} tournament hosted by {ORGANISERS} <!-- list the organisers here with their flags -->. It is the {ITERATION_ORDINAL} instalment of the {SERIES_TITLE}.
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2018-01-01/2018-01-28 |
+| Live drawings | 2018-01-29 ({DRAW_TIME}) |
+| Group stage |  |
+| Round of 16 |  |
+| Quarterfinals |  |
+| Semifinals |  |
+| Finals week 1 |  |
+| Finals week 2 |  |
+
+<!-- Use the YYYY-MM-DD format for dates e.g. 2018-01-01
+Use (HH:MM UTC+N) format for {DRAW_TIME} e.g. (12:00 UTC+07)
+ -->
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") |  |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") |  |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") |  |
+| *4th place* |  |
+
+![](img/badge.jpg "{SHORT_TITLE} winner badge") <!-- remove if not available -->
+
+## Organisation
+
+The {TITLE} is run by various community members.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Manager |  |
+| Mappool selector |  |
+| Streamer |
+| Commentator |  |
+| Designer |
+| Referee |
+| Statistician |  |
```

Comment:
It probably wouldn't break if not changed, but it should be included for consistency.

```suggestion
| Position | Member(s) |
| :-- | :-- |
| Manager |  |
| Mappool selector |  |
| Streamer |  |
| Commentator |  |
| Designer |  |
| Referee |  |
| Statistician |  |
```